install.sh

```diff
@@ -156,10 +162,10 @@ if $nonroot; then
     sed -i -e "s#/var/lib/scylla#$rprefix#g" "$rsysconfdir"/scylla-jmx
     sed -i -e "s#/etc/scylla#$rprefix/etc/scylla#g" "$rsysconfdir"/scylla-jmx
     sed -i -e "s#/opt/scylladb/jmx#$rprefix/jmx#g" "$rsysconfdir"/scylla-jmx
-    if check_usermode_support; then
+    if ! $supervisor && check_usermode_support; then
         systemctl --user daemon-reload
     fi
     echo "Scylla-JMX non-root install completed."
-elif ! $packaging; then
+elif ! $supervisor && ! $packaging; then
     systemctl --system daemon-reload
 fi
```

Comment:
I see it doesn't copy any supervisor config files. I think they should be moved to this package, just like the systemd integration is here.